src/styles/BEM-blocks/card.scss

```diff
@@ -0,0 +1,140 @@
+@import "../utils/_variables.scss";
+
+.card {
+  /* display flex ect. */
+  box-sizing: border-box;
+  width: $card-bg-size-w;
+  height: $card-bg-size-h;
+  border-radius: $card-border-radius;
+  padding: 32px 16px 16px;
+
+  /* color design */
+  border: 1px solid $card-border-color;
+  background: $card-bg-light-color;
+  font-family: "Roboto", Helvetica, sans-serif;
+
+  /* animation -wyszczególnienie nie działało- */
+  transition-property: transform;
+  transition-duration: 300ms;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
+  }
+
+  &__title {
+    margin-top: 36px;
+    color: $card-text-main-color;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+
+    /* animation */
+    transition-property: color;
+    transition-duration: 300ms;
+  }
+
+  &__code {
+    margin-top: 4px;
+    color: $card-text-side-color;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__rating {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: column;
+    flex-shrink: 0;
+
+    /* deisgn */
+    color: $card-text-main-color;
+    text-align: right;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    color: $card-text-side-color;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-num {
+    color: $card-text-main-color;
+    text-align: right;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buy-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-shrink: 0;
+    width: $card-button-w;
+    height: $card-button-h;
+    border-radius: $card-border-radius;
+    margin-top: 16px;
+    box-sizing: border-box;
+
+    /* design */
+    background-color: $card-blue-accent;
+    color: $card-bg-light-color;
+    text-align: center;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    /* animation */
+    transition-duration: 300ms;
+    transition-property: color, background-color;
+
+    &:hover {
+      flex-shrink: 0;
+      width: $card-button-w;
+      height: $card-button-h;
+      border-radius: $card-border-radius;
+
+      /* design */
+      border: 1px solid $card-blue-accent;
+      background-color: $card-bg-light-color;
+      color: $card-blue-accent;
+    }
+  }
+
+  & > a {
+    text-decoration: none;
```

Comment:
you can actually set that in style reset - it's rarely the case to want `text-decoration` to be anything other than `none` 😄